cfg/sources.js

```diff
@@ -0,0 +1,34 @@
+export const SOURCES = Object.freeze({
+    COLD_EMAIL: 'COLD_EMAIL',
+    LINKEDIN_DM: 'LINKEDIN_DM', 
+    LINKEDIN_POST: 'LINKEDIN_POST', 
+    TRELLUS_WEBSITE: 'TRELLUS_WEBSITE',
+    MANUAL: 'MANUAL'
+  });
+
+export const TrackerType = Object.freeze({
+    INSTALL: 'INSTALL',
+    CLICK: 'CLICK'
+});
+
+export const SOURCE_TO_LEARN_LINK_REGEX = Object.freeze({
+    [SOURCES.COLD_EMAIL]: new RegExp('learn-now', 'i'),
+    [SOURCES.LINKEDIN_DM]: new RegExp('learn-more', 'i'),
+    [SOURCES.LINKEDIN_POST]: new RegExp('learn-about', 'i'),
+});
+
+export const SOURCE_TO_INSTALL_LINK_REGEX = Object.freeze({
+    [SOURCES.COLD_EMAIL]: new RegExp('try-now', 'i'),
+    [SOURCES.LINKEDIN_DM]: new RegExp('get-trellus', 'i'),
+    [SOURCES.LINKEDIN_POST]: new RegExp('try-trellus', 'i'),
+    [SOURCES.TRELLUS_WEBSITE]: new RegExp('get-started', 'i'),
+    [SOURCES.MANUAL]: new RegExp('install-now', 'i')
+  });
+
+
+export const SOURCE_PARAM_STORAGE = 'source'
+export const REFERRER_PARAM_STORAGE = 'referrer'
+export const REFERRER_PARAM_URL = 'ref'
```

Comment:
can we get some short comments here on what these are? (e.g. some are local storage keys , some are query params that will be looked for, in all cases it'd be nice to know what sort of object would be associated with the key / param)